shuup/front/apps/carousel/locale/fi_FI/LC_MESSAGES/django.po

```diff
@@ -205,6 +223,21 @@ msgid ""
 "banner boxes."
 msgstr "Tätä lomaketta käytetään myös bannerilaatikoiden määrittelyyn. Merkatut kentät vaikuttavat vain bannerilaatikoihin."
 
+msgid "Adding the carousel to your site"
+msgstr "Karusellin lisääminen sivustolle"
+
+msgid ""
+"To add a carousel to your site, you must edit the page using the placeholder"
+" editor. To do this, go to your site and click the 'Edit page' on the page "
+"you want to add the carousel banner to. Then, select the placeholder to "
+"change and add a new plugin row and from the plugin dropdown and select "
+"'Carousel Plugin'. From 'Carousel' dropdown options choose the carousel you "
+"want to display. As a final step, save the plugin and publish changes."
+msgstr "Lisätäksesi karusellin sivustollesi sinun tulee muokata halutun sivun sisältöaluetta. Mene kaupan puolelle sivulle, jolle haluat lisätä karusellin ja klikkaa 'Muokkaa Sivua'. Valitse sisältöalue ja lisää lisäosa. Lisäosa listasta valitse 'Karuselli lisäosa'.  'Karuselli' valinnasta valitse karuselli, jonka haluat näyttää sisältöalueessa. Lopuksi tallenna liitännäinen ja julkaise muutokset."
```

Comment:
Add line changes like on the `msgid` above

also, the text could be something like this 
```
"Lisätäksesi karusellin sivustollesi, tulee sinun muokata halutun sivun sisältöaluetta seuraavasti; "
"mene kaupassa sivulle, jolle haluat lisätä karusellin ja klikkaa 'Muokkaa Sivua'. "
"Valitse tämän jälkeen sisältöalue sekä lisää lisäosa. Valitse lisäosien listasta "
"'Karuselli lisäosa'. Valitse 'Karuselli'-valinnasta karuselli, jonka haluat näyttää "
"sisältöalueessa. Tallenna lopuksi liitännäinen ja julkaise muutokset."
```